de.fu_berlin.inf.dpp.whiteboard/src/de/fu_berlin/inf/dpp/whiteboard/gef/part/ElementRecordPart.java

```diff
@@ -55,9 +57,24 @@ protected void createEditPolicies() {
             new ElementModelDeletePolicy());
     }
 
+    /**
+     * @return supported children models that can be rendered by the original
```

Comment:
What is the original whiteboard ? I am sure you mean the Eclipse Whiteboard that uses the GEF